documentation/api/notation.rst

```diff
@@ -287,17 +287,18 @@ of the sensor's resolution, e.g. hourly or daily values if the sensor's resoluti
 Sources
 -------
 
-Requests for data may limit the data selection by specifying a source, for example, a specific user.
-Account roles are also valid source selectors.
-For example, to obtain data originating from either a meter data company or user 42, include the following:
+Requests for data may filter by source. FlexMeasures keeps track of the data source (the data's author, for example, a user, forecaster or scheduler belonging to a given organisation) of time series data.
+For example, to obtain data originating from data source 42, include the following:
 
 .. code-block:: json
 
     {
-        "sources": ["MDC", "42"],
+        "source": 42,
     }
 
-Here, "MDC" is the name of the account role for meter data companies.
+Data source IDs can be found by hovering over data in charts.
```

Comment:
Is it an idea to make an issue for a page that shows available data sources?

Reply:
#546